### PR TITLE
Fix commands that do not pass the specified tests.

### PR DESCRIPTION
**Pull request by DaeguDude**
### Updated
#88 Updated `callbacks` command to be its own group
callbacks

#89 Updated `cqg` command to be its own group
cqg

#90 Updated `help`, `code` commands to be its own group
help

#94 Updated all commands to pass the tests
simpleCommands

#95 Updated `time` command to be its own group
time

### CODE REVIEW

**Comment by DaeguDude** [comment]
@twalton83 I've updated to fix the some of the issues.
I wasn't so sure about the git workflow, for example when to make commits or etc, so I just made a commit every time I updated the each command.

Please let me know if there's some I could have improved!

File: botCommands/simpleCommands.test.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ describe('/smart', () => {
       ['/smrt'],
       ['/smarts'],
       ['```function("/smart", () => {}```'],
-      ['/smart'],
+      ['/smarta'],
```
Comment (twalton83) [inline botCommands/simpleCommands.test.js:79]: Good catch here, thank you.

File: botCommands/simpleCommands.test.js [hunk 2]
```diff
@@ -431,7 +431,7 @@ describe('/peen', () => {
       ['/pee'],
       ['/peens'],
       ['```function("/peen", () => {}```'],
-      ['/peen'],
+      ['/peena'],
```
Comment (twalton83) [inline botCommands/simpleCommands.test.js:434]: Great catches for all of these!

**Review by twalton83 — APPROVED** [review]
Thanks for assisting here.

**Comment by DaeguDude** [comment]
#93  Updated `shrug` command to be its own group
Screen Shot 2021-01-29 at 3 18 58 PM

**Comment by twalton83** [comment]
I should have been more clear that these should have been separate PRs, my bad! Just make a separate PR moving forward if it’s for a separate issue.

**Comment by DaeguDude** [comment]
Oops, my bad! Should I close this pull request and make each one for each issue?

**Comment by twalton83** [comment]
No worries, just a heads up moving forward!

**Comment by DaeguDude** [comment]
Okay, I will make sure of it next time :) Thanks for the heads up!

**Comment by twalton83** [comment]
Thanks again for doing all of these!

**Pull request merged**